### PR TITLE
ENYO-6308: Add QA sample for Spinner with Input

### PR DESCRIPTION
**Pull request by LucieRoy**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: packages/sampler/stories/qa/Spinner.js [hunk 1]
```diff
@@ -15,6 +17,38 @@ const
...
+class FocusOnSpinner extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			txt: '',
+			isLoading: false
+		};
+	}
+
+		handleChange = (e) => {
```
Comment (webOS101) [inline packages/sampler/stories/qa/Spinner.js]: Why isn't eslint complaining about this indentation? :/

File: packages/sampler/stories/qa/Spinner.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import Spinner from '@enact/moonstone/Spinner';
 import Button from '@enact/moonstone/Button';
-import ri from '@enact/ui/resolution';
+import Input from '@enact/moonstone/Input';
+import Spinner from '@enact/moonstone/Spinner';
 import React from 'react';
+
```
Comment (webOS101) [inline packages/sampler/stories/qa/Spinner.js]: Please remove this space.

File: packages/sampler/stories/qa/Spinner.js (hunk 1 above)
Comment (webOS101) [inline packages/sampler/stories/qa/Spinner.js]: Please fix indentation of these to be one tab, not two.

File: packages/sampler/stories/qa/Spinner.js [hunk 3]
```diff
@@ -15,6 +17,38 @@ const
...
+		handleDeactivate = () => {
+			this.setState({
+				isLoading: true
+			});
+		}
+
+		render () {
+			return (
+				<div>
+					<Button>open</Button>
```
Comment (webOS101) [inline packages/sampler/stories/qa/Spinner.js]: Does this `Button ` serve a purpose?  Can you add some text to describe how this test is supposed to work?

Comment (LucieRoy) [reply packages/sampler/stories/qa/Spinner.js]: Removed

File: packages/sampler/stories/qa/Spinner.js (hunk 1 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa/Spinner.js]: Done

File: packages/sampler/stories/qa/Spinner.js (hunk 2 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa/Spinner.js]: Done

**Pull request merged**